cmd/gaia/init/testnet.go

```diff
@@ -82,7 +81,8 @@ Example:
 		client.FlagChainID, "", "genesis file chain-id, if left blank will be randomly created",
 	)
 	cmd.Flags().String(
-		flagMinimumFees, fmt.Sprintf("1%s", stakingtypes.DefaultBondDenom), "Validator minimum fees",
+		server.FlagMinGasPrices, fmt.Sprintf("1%s", stakingtypes.DefaultBondDenom),
+		"Minimum disparate gas prices validator will accept for transactions",
```

Comment:
I think this could be more descriptive... "disparate gas prices" is not clear

Reply:
"At least one fee denom amount must be greater than or equal to any of the denom amount specified by this flag"

Reply:
Updated.